### ISSUE TITLE
Auto import has_part entities for complexes

### ISSUE THREAD

**Issue by ValWood** [geneontology/noctua#905]
It would be great if all of the subunits of the complex were instantiated from the data in the GPAD?GPI file
(it's really tedious to have to enter them using has_part)

**Comment by ValWood** [comment]
This would be really great for efficiency

**Comment by sjm41** [comment]
> It would be great if all of the subunits of the complex were instantiated from the data in the GPAD?GPI file

I agree. As long as the imported subunit list could then be edited (to add missing members, remove extraneous members, refine context-specific composition), then this should work well.

Maybe pulling in complex composition based on existing CC annotations should be optional, since its utility would depend on how good your CC annotations terms are....

**Comment by vanaukenk** [comment]
Closing this in lieu of a duplicate issue in the VPE tracker:

ithub.com/geneontology/noctua-visual-pathway-editor/issues/82
